src/protocols/secure_channel/CheckinMessage.h

```diff
@@ -74,13 +74,15 @@ class DLL_EXPORT CheckinMessage
      * @param[in]       hmac128KeyHandle  Key handle with which to verify the received nonce in the check-in payload (using HMAC).
      * @param[in]       payload           The received payload to decrypt and parse
      * @param[out]      counter           The counter value retrieved from the payload
+     *                                    If an error occurs, no value will be set.
      * @param[in,out]   appData           The optional application data decrypted. The input size of appData must be at least the
-     * size of GetAppDataSize(payload) + sizeof(CounterType), because appData is used as a work buffer for the decryption process.
-     * The output size on success will be GetAppDataSize(payload).
+     *                                    size of GetAppDataSize(payload) + sizeof(CounterType), because appData is used as a work
+     * buffer for the decryption process. The output size on success will be GetAppDataSize(payload).
```

Comment:
The indent here is sort of odd...

Reply:
I agree but the restyler won't play nice and i have no idea how to work around this...

Reply:
Trailing white space was confusing the restyler. fixed the indenting